conf/knnaas_log_config.yaml

```diff
@@ -18,7 +18,7 @@ handlers:
         class: logging.handlers.RotatingFileHandler
         level: INFO
         formatter: simple
-        filename: test/knnaas_info.log
+        filename: /var/log/Cuppa/knnaas/knnaas_info.log
```

Comment:
These are not production paths. They've been moved to a local path, so setup is easy and convenient for the end user.  Test directory does not exist now. Let it log to the current directory, so if the services are started from the Cuppa directory, then logs will be written to Cuppa
So: filename: knnaas_info.log